plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/query/HudiReadOptimizedDirectoryLister.java

```diff
@@ -55,7 +61,10 @@
 
     private List<String> hivePartitionNames;
 
+    private List<String> commitTimeRanges;
+
     public HudiReadOptimizedDirectoryLister(
+            ConnectorSession session,
```

Comment:
We don't need the entire `ConnectorSession`. Only timestamp format should be sufficient.

Reply:
fixed